src/modules/custom.cpp

```diff
@@ -130,6 +132,19 @@ void waybar::modules::Custom::parseOutputRaw()
   }
 }
 
+bool waybar::modules::Custom::isInteger(const std::string& n)
```

Comment:
It is not necessary the JSON format supports numbers so just do isUInt and AsUInt :)

Reply:
I have tried this but it nerver triggered, but after having a second look at it, my json was not vorrectly formated. I'll push a commit in a sec.